### ISSUE TITLE
Code blocks in the api documentation aren't styled correctly

### ISSUE THREAD

**Issue by xt0rted** [StylishThemes/GitHub-Dark#594]
* **Browser**: Chrome 64
* **Operating System**: Win10
* **URL**: https://developer.github.com/v3/
* **Screenshot**:

![image](https://user-images.githubusercontent.com/831974/37443044-04350a3e-27e0-11e8-9566-e6b87fce6499.png)

* **HTML of the section where the issue occurs**:

````html
<li>
<p>Sending invalid JSON will result in a <code>400 Bad Request</code> response.</p>

<pre><code>HTTP/1.1 400 Bad Request
Content-Length: 35

{"message":"Problems parsing JSON"}
</code></pre>
</li>
````

```html
<tr>
<td><code>missing</code></td>
<td>This means a resource does not exist.</td>
</tr>
```

**Comment by Mottie** [comment]
Hi @xt0rted!

I'm not seeing this issue. Do you have another style that's interfering?

**Comment by Poorchop** [comment]
I'm having this on my end too. I disabled my other styles to see if they were the problem but they weren't. Even the code blocks have some weird styling applied to them - my syntax highlighting scheme is Monokai and the background color seems to be applied, but I don't think that dark blue or burnt orange are part of the color scheme.

![screenshot](https://user-images.githubusercontent.com/5256208/37492815-205bfe90-2879-11e8-9899-acbcf354b6e6.png)

**Comment by xt0rted** [comment]
@Mottie I don't have any other styles applied for github right now

I'm using Monokai as well and that's what the code blocks look for me.

**Comment by Poorchop** [comment]
At least for the white boxes, I think that there needs to be a selector for `body.api code` that I currently don't see anywhere in the style.

**Comment by the-j0k3r** [comment]
All OK here in Firefox 59.0 1 Windows 10

![test](https://user-images.githubusercontent.com/31389848/37709228-7557424c-2d01-11e8-8f9b-11f1134b37f3.gif)

**Comment by Mottie** [comment]
Yep, I've been working on an update. I just got distracted at some point 😉

**Comment by Mottie** [comment]
All fixed! Please don't forget that the Jupyter, not the GitHub, theme is being used on the api pages.

**Comment by the-j0k3r** [comment]
![](https://proxy.duckduckgo.com/iu/?u=http%3A%2F%2Fmedia.riffsy.com%2Fimages%2F2a5662a25de3a724f5ac08965d275233%2Ftenor.gif&f=1)
